recipes/parasail-python/meta.yaml

```diff
@@ -21,6 +24,7 @@ requirements:
     - pip
     - setuptools
     - {{ compiler('c') }}
```

Comment:
In build should be only the compilers autoconf, automake, libtool and m4 I think.

Reply:
@bgruening That makes sense, thanks for pointing that out. I've updated the recipe to leave only the build tools under build, and moved the rest to host. I've also removed numpy as a build-time requirement.